src/agl/WebRuntimeAGL.h

```diff
@@ -10,8 +10,11 @@
 #include "WebRuntime.h"
 #include <libxml/parser.h>
 
+#include "AglShell.h"
+
 class LibHomeScreen;
 
+#if 0
```

Comment:
We don't #if 0, we just remove code on refactors (git will still keep previous version if needed).

Reply:
Probably some left overs I haven't squashed. Will remove.